optimum_benchmark/scenarios/energy_star/preprocessing_utils.py

```diff
@@ -32,7 +58,7 @@ def tokenize_function(examples):
             examples[config.text_column_name],
             padding=padding,
             truncation=config.truncation,
-            max_length=config.max_length if config.max_length != -1 else None,
+            max_length=getattr(pretrained_config, "max_position_embeddings", 512),
```

Comment:
using `model_shapes` (normalized dictionary of model variables) makes more sense here.